app/services/weaviate_vectordb_client.py

```diff
@@ -161,7 +161,7 @@ def create_schema(self, class_name: str, class_obj: dict):
             logger.info(f"Entering create_schema")
             # wrap with try catch to avoid error if the class already exists
             try:
-                if not self.client.schema.contains(class_obj):            
+                if not self.client.schema['classes'].contains(class_obj):            
```

Comment:
@YanDav78 
It might not be needed
we saw that the class and objects fail if the scema isnt found but then when trying to create it fails.
It is possible that this is because class has changed. It is scary though that any schema change we make to the class will clean all current objects. I thought maybe the check is off because the classes are under the classes field but it might not be the cause. Either way we need to protect ourselves from destructive (right now any) schema changes.
@itamar-cord investigated

Reply:
The condition I have is correct and was made on purpose. When we have schema changes we must recreate for now.
And I agree that we can't do that when we go live. But for now it must be done. In the future we will update the schema instead of recreating it.